scripts/prepublish

```diff
@@ -11,6 +11,16 @@ if ! npm outdated --long; then
   fi
 fi
 
+source 'node_modules/sanctuary-scripts/functions'
+
+pkg="$(get repo-name)"
+files=($(get source-files))
+
+sed --in-place "s/$pkg@0.0.0/$pkg@$VERSION/" "${files[@]}"
+sed --in-place "s/$pkg@$PREVIOUS_VERSION/$pkg@$VERSION/" "${files[@]}"
```

Comment:
Did you consider combining the patterns?

```suggestion
sed -E --in-place "s/$pkg@(0.0.0|$PREVIOUS_VERSION)/$pkg@$VERSION/" "${files[@]}"
```

Should substitutions be performed globally? Currently only the first occurrence of the pattern on each line will be replaced.

Reply:
> Did you consider combining the patterns?

I tried and failed. I guess the `-E` is needed for groupings.

> Should substitutions be performed globally?

Hm, that does sounds like it would be better, although now it's just being used to update URLs, each on their own line like `//. [foo]: foo.example.com/fluture@1.2.3`.

Reply:
I'll look into this later: https://github.com/fluture-js/fluture-template/issues/11